worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -35,7 +35,7 @@
     "worldedit.setbiome.warning": "You may have to re-join your game (or close and re-open your world) to see changes.",
 
     "worldedit.drawsel.disabled": "Server CUI disabled.",
-    "worldedit.drawsel.enabled": "Server CUI enabled. This only supports cuboid regions, with a maximum size of 32x32x32.",
+    "worldedit.drawsel.enabled": "Server CUI enabled. This only supports cuboid regions, with a maximum size of {0}x{0}x{0}.",
```

Comment:
I wonder if we should make this `{0}x{1}x{2}` in case the dimensions become different in the future.

Reply:
I somewhat doubt it'd ever get to the point where it's different in the different axis

Reply:
I'd still like to make this change, I think it's better overall. It doesn't really harm anything, and it allows forward-compatibility.